chain/sync.go

```diff
@@ -160,6 +160,37 @@ func NewSyncer(ds dtypes.MetadataDS,
 	return s, nil
 }
 
+// NewSyncerFollower creates a new Syncer object that the follower node uses
+func NewSyncerFollower(ds dtypes.MetadataDS,
+	sm *stmgr.StateManager,
+	exchange exchange.Client,
+	syncMgrCtor SyncManagerCtor,
+	connmgr connmgr.ConnManager,
+	self peer.ID,
+	beacon beacon.Schedule,
+	gent Genesis,
+	consensus consensus.Consensus) (*Syncer, error) {
+
+	s := &Syncer{
```

Comment:
We shouldn't run the chain syncer on follower nodes - the follower node should be following ChainHead from syncer node(s).

Reply:
If i do not start up the Chain syncer, I get an error in the Hello module, line 85:



```
if hmsg.GenesisHash != hs.syncer.Genesis.Cids()[0] {
                log.Debugf("other peer has different genesis! (%s)", hmsg.GenesisHash)
                _ = s.Conn().Close()
                return
}
```

I'm investigating disabling the Hello module in Follower mode as well

Reply:
That seems to work now!

Reply:
#e8e96a267